### ISSUE TITLE
Share Items via Bluetooth(?)

### ISSUE THREAD

**Issue by vetinari** [simonoppowa/OpenNutriTracker#81]
When preparing a meal, I'd like to share the ingredients with my wife, so she doesn't have to go through the same steps again. Maybe in combination with #66 this would be most helpful.

**Comment by freshLobster** [comment]
I agree, this needs to be a feature. even if the data is just shared in a text message. Often times I have to create a custom meal item and my girlfriend has to manually enter the same details of the meal. It would be ideal to have a way to share a meal or even just individual items that could just add the items to the other person's tracker. maybe a scan QR function added to the add item search?

**Comment by jordan-lee-code** [comment]
I have addressed the QR code sharing in the feature in #326
